web/Web/web/judgment/views.py

```diff
@@ -56,6 +56,7 @@ def post(self, request, *args, **kwargs):
             found_ctrl_f_terms_in_title = self.request_json.get(u"found_ctrl_f_terms_in_title", None)
             found_ctrl_f_terms_in_summary = self.request_json.get(u"found_ctrl_f_terms_in_summary", None)
             found_ctrl_f_terms_in_full_doc = self.request_json.get(u"found_ctrl_f_terms_in_full_doc", None)
+            len_docs = self.request_json.get(u"len_docs", None)
```

Comment:
Shouldn't the request from the client side have `len_docs` somewhere? I don't see changes to any JS, probably forgot to commit?

Reply:
Have the static files been moved?
Is the correct place for js `web/Web/staticfiles` or `web/Web/web/static`?

Reply:
`web/Web/web/static` is the correct one. The `staticfiles` folder is automatically generated by Django before it runs.